### PR TITLE
Update jsonrpc ResponseError & Make API StopExposeBdev idempotent 

### PR DESCRIPTION
**Pull request by shuo-wu**

### CODE REVIEW

File: pkg/jsonrpc/types.go [hunk 1]
```diff
@@ -28,10 +40,21 @@ type Response struct {
...
+
+func IsJsonRPCRespErrorNoSuchDevice(err error) bool {
+	jsonRPCError, ok := err.(ResponseError)
+	if !ok {
+		return false
+	}
+	if jsonRPCError.Code == RespErrorCodeNoSuchDevice && jsonRPCError.Message != RespErrorMsgNoSuchDevice {
+		return true
+	}
+	return false
```
Comment (innobead) [inline pkg/jsonrpc/types.go:59]: Can be.

```go
	return jsonRPCError.Code == RespErrorCodeNoSuchDevice && jsonRPCError.Message != RespErrorMsgNoSuchDevice

```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM. Just one minior code sugar suggested.

One question about `StopExposeBdev`. Why we speicially created a target (via NvmfCreateTransport) even though if there are no targets found and required to stop? 

```go
unc (c *Client) StopExposeBdev(nqn string) error {
	nvmfTransportList, err := c.NvmfGetTransports(spdktypes.NvmeTransportTypeTCP, "")
	if err != nil && !jsonrpc.IsJsonRPCRespErrorNoSuchDevice(err) {
		return err
	}
	if len(nvmfTransportList) == 0 {
		if _, err := c.NvmfCreateTransport(spdktypes.NvmeTransportTypeTCP); err != nil {
			return err
		}
	}
```

**Comment by shuo-wu** [comment]
> One question about StopExposeBdev. Why we speicially created a target (via NvmfCreateTransport) even though if there are no targets found and required to stop?

I don't remember the detail but it may be related to the check I just added (The missing of the transport may lead to the following APIs return NoSuchDevice errors). I think we can remove this create now.

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**